ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/AbstractEngineForkchoiceUpdated.java

```diff
@@ -177,7 +177,8 @@ public JsonRpcResponse syncResponse(final JsonRpcRequestContext requestContext)
                     payloadAttributes.getTimestamp(),
                     payloadAttributes.getPrevRandao(),
                     payloadAttributes.getSuggestedFeeRecipient(),
-                    withdrawals));
+                    withdrawals,
+                    Optional.empty()));
```

Comment:
Do we need to pass deposits in here? If we're preparing a payload (creating a block) then won't we parse the transactions and create the deposit objects ourselves later on in the block creation code?